src/components/course-settings/delete-period.cjsx

```diff
@@ -72,18 +84,13 @@ module.exports = React.createClass
         {deleteButton if @isPeriodEmpty()}
       </div>
 
-
-
     </BS.Modal>
 
   render: ->
-    <BS.OverlayTrigger
-      ref='overlay'
-      rootClose={true}
-      trigger='click'
-      overlay={@renderForm()}>
-        <BS.Button bsStyle='link' className='edit-period'>
-          <i className='fa fa-trash-o' />
-          Delete <CourseGroupingLabel courseId={@props.courseId}/>
-        </BS.Button>
-    </BS.OverlayTrigger>
+    <span>
```

Comment:
Can you add a `class="-something"` to this element so it is easier to identify/find later on? There are other places in this diff that also add a `<span>`.